packages/manager/modules/enterprise-cloud-database/src/card/card.html

```diff
@@ -0,0 +1,34 @@
+
+<div class="card"
+    data-ng-class="[$ctrl.class, {
+        'disabled': $ctrl.disabled,
+        'selected': $ctrl.selected,
+      }]"
+    data-ng-click="$ctrl.onClick({ modelValue: $ctrl.model })"
+    role="radio"
+    aria-checked="{{ $ctrl.selected }}"
+    data-ng-attr-id="{{ ::$ctrl.id }}"
+    data-ng-attr-name="{{ ::$ctrl.name }}">
+    <div class="card-body">
+        <div class="card-icon"
+            data-ng-if="$ctrl.bodyTranscluded"
+            data-ng-transclude="bodySlot">
+        </div>
+        <div class="card-icon" data-ng-if="!$ctrl.bodyTranscluded && $ctrl.image">
+            <img alt=""
+                aria-hidden="true"
+                role="presentation"
+                data-ng-if="$ctrl.isImgPath"
+                data-ng-src="{{ :: $ctrl.image }}">
+            <span aria-hidden="true"
+                role="presentation"
+                data-ng-class=":: $ctrl.image"
+                data-ng-if="!$ctrl.isImgPath"></span>
+        </div>
+        <span data-ng-if="!$ctrl.bodyTranscluded" data-ng-bind="$ctrl.label"></span>
+    </div>
+    <div class="card-footer"
+        data-ng-if="$ctrl.footerTranscluded"
+        data-ng-transclude="footerSlot">
+    </div>
+</div>
```

Comment:
This looks like a general component that could be in UI-Kit or an existing component that can be updated

Reply:
I said the same ✋